File: 59/Problem 2.c
#include <stdio.h>

int main(void) {
	int cases;
	int m, n;
	int h, t ,d;
	int flag;
	scanf("%d", &cases);
	while(cases--){
	    flag = 0;
	    scanf("%d%d", &m, &n);
	    while(m <= n){
	        h = m / 100;
	        t = m / 10;
	        d = m % 10;
	        if(h == 0 && t == d){
	            if(flag == 0){
	                flag = 1;
	            }
	            else{
	                printf(" ");
	            }
	            printf("%d", m);
	        }
	        else if(h == d){
	            if(flag == 0){
	                flag = 1;
	            }
	            else{
	                printf(" ");
	            }
	            printf("%d", m);
	        }
	        m++;
	    }
	}
	return 0;
}

File: 61/Problem 2.cs
using System;

namespace ConsoleApp1
{
    class Program
    {
        static void Main(string[] args)
        {
            int cases;
            int[] item = new int[] { 0, 17, 25 };
            int[] coin = new int[] { 50, 10, 5, 1 };
            int[] qty = new int[4];
            cases = int.Parse(Console.ReadLine());
            while(cases > 0)
            {
                string[] data = Console.ReadLine().Split(',');
                int money = int.Parse(data[0]);
                int mitem = item[int.Parse(data[1])];
                int mqty = int.Parse(data[2]);
                money = money - mitem * mqty;
                Array.Clear(qty, 0, qty.Length);
                for(int i = 0; i <= 3 && money != 0; i++)
                {
                    int div = (money / coin[i]);
                    if (div > 0)
                    {
                        money -= coin[i] * div;
                        qty[i] += div;
                    }
                }
                bool flag = false;
                for(int i = 3; i >= 0; i--)
                {
                    if(qty[i] != 0 && !flag)
                    {
                        Console.Write("Coin " + coin[i] + ":" + qty[i]);
                        flag = true;
                    }
                    else if(qty[i] != 0 && flag)
                    {
                        Console.Write(",Coin " + coin[i] + ":" + qty[i]);
                    }
                }
                Console.WriteLine();
                cases--;
            }
        }
    }
}
